sound/soc/sof/topology.c

```diff
@@ -1590,7 +1590,7 @@ static int sof_widget_load_pga(struct snd_soc_component *scomp, int index,
 	if (!volume)
 		return -ENOMEM;
 
-	if (le32_to_cpu(tw->num_kcontrols) != 1) {
+	if (!le32_to_cpu(tw->num_kcontrols)) {
```

Comment:
do we actually support more than 2 controls?
if no, what is the rationale for testing != 0

Reply:
see comments on the same matter above... if the fact is that there can't be more than 2 controls I'll change it back, I guess in practice it is at most 2, but I'm not aware of limitations otherwise.